gninasrc/lib/nngridder.h

```diff
@@ -176,7 +174,9 @@ class NNGridder {
             gpu_ligandAInfo(NULL), gpu_ligWhichGrid(NULL) {
     }
 
-    void initialize(const gridoptions& opt);
+    virtual ~NNGridder() {if (gmaker) delete gmaker;}
```

Comment:
make sure copy constructor is deleted or implemented properly to avoid copying pointer (or used shared ptr?)